cmd/notifier/config/config.toml

```diff
@@ -1,10 +1,12 @@
+num_main_shards = 3
```

Comment:
perhaps keep the camel case structure as in other projects?

Reply:
Actually, we are using them kinda peculiar in other projects.

These should be `snake_case`: https://stackoverflow.com/questions/51543501/naming-convention-for-keys-in-toml-file